_posts/2020-03-12-Steering-Group-executive-summary,-action-items,-and-minutes.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Steering Group executive summary, action items, and minutes
+author: Franklin Feingold
+display: true
+---
+
+# Steering Group executive summary, action items, and minutes
+
+Date: Thursday February 13, 2020
+Attended: Franklin Feingold, Melanie Ganz, Guiomar Niso, Robert Oostenveld, Kirstie Whitaker
+
+### Executive Summary
+
+We discussed the channels through which to share updates to the BIDS specification.
+An issue was opened to start crowdsourcing and collecting this information.
+We will be using this to share the release of v1.2.2 and to request the final rounds of feedback on the Genetic information extension.
```

Comment:
"an issue was opened" --> a GitHub issue? If yes, include number+link

We will be using **this** --> what do you mean with "this"? The issue? The info gained from responses to the issue?

Genetric information extension --> add BEP number and link